types/props.ts

```
@@ -0,0 +1,68 @@
+
+export interface ViewCounterProps {
+  slug: string;
+  hideText?: boolean;
+}
+
+export interface ReportViewProps {
+  slug: string;
+}
+
+export interface CardContentProps {
+  event: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)